setup.py

```diff
@@ -57,7 +57,7 @@ def run(self):
 
 
 setup(name='citrine',
-      version='0.71.0',
+      version='0.71.1',
```

Comment:
```suggestion
      version='0.72.1',
```

Reply:
@kroenlein Why make this a minor version bump instead of incrementing the revision number? Also, did you mean to suggest 0.72.0?

Reply:
Never mind. I see version 0.72.0 is the latest in master.